felix/fv/vxlan_test.go

```diff
@@ -558,11 +558,11 @@ var _ = infrastructure.DatastoreDescribe("_BPF-SAFE_ VXLAN topology before addin
 					}
 
 					// Disable VXLAN in Felix.
-					felixConfig, err := client.FelixConfigurations().Get(context.Background(), "default", options.GetOptions{})
-					Expect(err).NotTo(HaveOccurred())
+					felixConfig := api.NewFelixConfiguration() // Create a default FelixConfiguration
```

Comment:
nit: generally I try to avoid inline comments. In this particular case, the comment doesn't tell us anything the code doesn't already tell us, so can do without a comment altogether really.

Reply:
Noted, will do! Sorry about that